docs/version-2-upgrade.md

```diff
@@ -0,0 +1,385 @@
+  to   = module.kinesis_data_stream_access.aws_iam_role_policy_attachment.attach_datastream_access[0]  
+}
+```
+
+## Removed Input: `custom_external_id`
+This input is no longer supported, and the `external_id` of the AWS IAM Role declared by this module will always be a random UUID.
+
+## Input `policy_arns` has been removed
+This input is no longer supported, and any additional AWS IAM Policies should be attached directly to the AWS IAM Role outputted by the module.
+
+For example, given this previous configuration:
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 1.0"
+
+  environment = var.environment
+  
+  policy_arns = [aws_iam_policy.example.arn]
+}
+```
+
+An updated configuration would be:
+
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 2.0"
+
+  environment = var.environment
+}
+
+resource "aws_iam_role_policy_attachment" "iam_policy_attachment_example" {
+  policy_arn = aws_iam_policy.example.arn
+  role       = module.runtime_connector.sym_runtime_connector_role.name
+}
+```
+
+## Removed Output: `account_id`
+The `account_id` output has been removed. You may still access this information with the following data block:
+```terraform
+data "aws_caller_identity" "current" {}
+data.aws_caller_identity.current.account_id
+```
+
+## Removed Output: `settings`
+The `settings` output has been superseded by the new `sym_integration` output.
+
+## New Output: `sym_integration`
+Version 2.0.0 of the `runtime_connector` module now outputs a `sym_integration.runtime_context` by default. You may choose
+to use this integration instead of declaring one explicitly.
+
+For example, given this previous configuration:
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 1.0"
+
+  environment = var.environment  
+}
+
+resource "sym_integration" "runtime_context" {
+  type        = "permission_context"
+  name        = "runtime-${var.environment}"
+  external_id = module.runtime_connector.settings.account_id
+
+
+  settings = module.runtime_connector.settings
+}
+
+resource "sym_secrets" "this" {
+  type = "aws_secrets_manager"
+  name = "${var.environment}-secrets"
+
+  settings = {
+    context_id = sym_integration.runtime_context.id
+  }
+}
+```
+
+An updated configuration would be:
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 1.0"
+
+  environment = var.environment  
+}
+
+resource "sym_secrets" "this" {
+  type = "aws_secrets_manager"
+  name = "${var.environment}-secrets"
+
+  settings = {
+    context_id = module.runtime_connector.sym_integration.id
+  }
+}
+```
+
+Optionally, you may choose to use `moved` configuration blocks to migrate your Terraform state, instead of destroying and recreating your existing `sym_integration.runtime_context`:
```

Comment:
```suggestion
Optionally, you may also use `moved` configuration blocks to migrate your Terraform state, instead of destroying and recreating your existing `sym_integration.runtime_context`:
```